src/components/StickerTemplate.js

```diff
@@ -20,37 +20,41 @@ import defensa from "../../assets/appAssets/test/defensa.png";
 import medioCentro from "../../assets/appAssets/test/medioCentro.png";
 import arquero from "../../assets/appAssets/test/arquero.png";
 
+// se borra luego esto es para tamaños
+import jugador from "../../assets/appAssets/test/ksa/ksa_5.png"
+
 const pos = '../assets/appAssets/test/'
 const { width } = Dimensions.get('window')
 
+/* cambiar luego de subir a online SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS */
+import Constants from 'expo-constants';
+/* SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS */
+
 export default function StickerTemplate({ sticker }) {
-    // const [id, playerName, country, position, img, height, weight] = sticker
-    // console.log(jugador, bandera, nombre, height, weight, position)
-    console.log(sticker.img)
-    // let ruta = pos + position + '.png'
     return (
         <View style={styles.container}>
             <ImageBackground
                 source={fondo}
                 style={styles.marco}
             >
-                <View style={{ alignItems: 'center' }}>
+                <View style={{ alignItems: 'center', overflow: 'hidden' }}>
                     <Image
-                        source={{uri:sticker.img}}
+                        // source={jugador}
+                        source={{ uri: sticker.img }} //cambiar a activo cuando ya este la BD
```

Comment:
```suggestion
                        source={{ uri: sticker.img }}
```
Hay que cambiar que 🤔 ?